### DISCUSSION TITLE
Xp-Pen Starg640 not detected

### DISCUSSION THREAD

**Discussion by Elwoko** [OpenTabletDriver/OpenTabletDriver#2375] [Q&A] [answered]
Hello,
I used to use this driver a lot but ever since the latest changes I can never get it to detect my tablet (Xp-PenSg640) for some reason
I tried going on other discussions to figure it out but I'm either small brain or it doesn't work for my tablet
I'm on windows 10 if it helps
thanks :)

**Comment by jamesbt365** [comment, marked as answer]
Could you give a diagnostic? help > export diagnostic > add a .txt to the end of the file name and upload here

**Reply by Elwoko** [reply]
I dug around a little bit and it turns out for whatever reason my computer itself wasn't detecting my tablet even though the button lit up when I pressed my pen on it
After a few restarts and reinstalls I got it to work in the end
All I did differently is that I didn't install the NET 6 Desktop Runtime x64 but i doubt that has anything to do with anything
Anyways I'm embarrassed now for not trying a little harder before and I apologize for wasting your time 
Thank you for your help though I didn't expect an answer so quickly

**Reply by jamesbt365** [reply]
No worries!
